PSMSVirtualDeviceManager/StartPage/UCStartPage.vb

```diff
@@ -492,18 +492,23 @@ Public Class UCStartPage
     End Sub
 
     Public Sub StopService()
```

Comment:
This looks quite messy, this can be cleaned up a little more.
Such as:

```visual-basic-.net
Dim mProcesses As Process()
[...]
mProcesses = Process.GetProcessesByName("PSMoveService")
For Each mProcess In mProcesses
``` 
to
```visual-basic-.net
For Each mProcess In Process.GetProcessesByName("PSMoveService")
```